gensim/models/word2vec.py

```diff
@@ -26,39 +28,64 @@
 >>> model.save(fname)
 >>> model = Word2Vec.load(fname)  # you can continue training with the loaded model!
 
-The model can also be instantiated from an existing file on disk in the word2vec C format::
+The word vectors are stored in a KeyedVectors instance in model.wv. This separates the read-only word vector lookup operations in KeyedVectors from the training code in Word2Vec.
+
+  >>> model.wv['computer']  # numpy vector of a word
+  array([-0.00449447, -0.00310097,  0.02421786, ...], dtype=float32)
+
+The word vectors can also be instantiated from an existing file on disk in the word2vec C format as a KeyedVectors instance::
+
+NOTE: It is impossible to continue training the vectors loaded from the C format because the binary tree is missing.
```

Comment:
Not just the binary tree (which is only used in `hs` mode), but the hidden-weights and vocabulary-frequency information are missing.